### ISSUE TITLE
 Cannot find module '@angular/cdk/a11y'.

### ISSUE THREAD

**Issue by Navyasri-Gurram** [chipto/context-menu#1]
ERROR in node_modules/ngx-contextmenu/lib/contextMenu.item.directive.d.ts(1,31): error TS2307: Cannot find module '@angular/cdk/a11y'.
node_modules/ngx-contextmenu/lib/contextMenu.service.d.ts(1,60): error TS2307: Cannot find module '@angular/cdk/overlay'.
node_modules/ngx-contextmenu/lib/contextMenuContent.component.d.ts(2,28): error TS2307: Cannot find module '@angular/cdk/overlay'.

Am getting this error after installing ngx-contextmenu package can any one help?
Thanks in advance

**Comment by zikaari** [comment]
Please open an issue in https://github.com/isaacplmann/ngx-contextmenu instead.
